llvm/test/CodeGen/AArch64/insertzero.ll

```diff
@@ -0,0 +1,27 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
```

Comment:
Do the existing tests cover all checks in the code? Might be good to at least have some tests with different vector sizes, including > 128 bits and odd number of elements.